.gitignore

```diff
@@ -1,4 +1,5 @@
 # custom
+cluster.yaml
 secrets.cfg
```

Comment:
Can we remove the secrets.cfg as part of this?

Reply:
? secrets is the most important one to be here

Reply:
Sorry, I meant to imply that I thought this was going to change to secrets.yaml, but I wasn't sure if it was part of this larger change.